roles/reset/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+
+- name: reset | stop services
+  service: name={{item}} state=stopped
+  with_items:
+    - kubelet
+    - etcd
+  failed_when: false
+
+- name: reset | remove services
+  file: path="/etc/systemd/system/{{item}}" state=absent
```

Comment:
Please use long format.
Also, it should be path: "/etc/systemd/system/{{ item }}.service"